runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/environment/DockerEnvironmentFactory.java

```diff
@@ -120,10 +125,10 @@ public RemoteEnvironment createEnvironment(Environment environment) throws Excep
     String provisionEndpoint = provisioningServiceServer.getApiServiceDescriptor().getUrl();
     String controlEndpoint = controlServiceServer.getApiServiceDescriptor().getUrl();
 
-    List<String> volArg =
-        ImmutableList.of(
-            // NOTE: Host networking does not work on Mac, but the command line flag is accepted.
-            "--network=host");
+    List<String> volArg = new ArrayList<>();
```

Comment:
Any reason not to use ImmutableList.Builder?

Reply:
Yes, much nicer. Done.